client/nginx.go

```diff
@@ -1052,6 +1087,15 @@ func (client *NginxClient) GetNginxInfo() (*NginxInfo, error) {
 	return &info, nil
 }
 
+func (client *NginxClient) getCaches() (*Caches, error) {
```

Comment:
Could you make this method exported? this is for consistency, as the other similar methods are exported. Could you also add a doc string for it?